ext/fast_hamming.c

```diff
@@ -1,10 +1,13 @@
 #include "ruby.h"
 
+// Don't allow the number of pairs collected to exceed MAX_PAIRS.
+const int MAX_PAIRS = 1250000;
```

Comment:
I think you should include some context about the choice for this number in the comments / rough expectations.